thalerj/openbmctool.py

```diff
@@ -0,0 +1,1490 @@
+#!/usr/bin/python3
+
+'''
+#================================================================================
+#
+#    openbmctool.py
+#
+#    Copyright IBM Corporation 2015-2017. All Rights Reserved
+#
+#    This program is licensed under the terms of the Eclipse Public License
```

Comment:
See the note about Apache 2 in my overview comment. Separately, i think we should start using SPDX license descriptors for the repo: see https://spdx.org/using-spdx